mlir/include/mlir/Dialect/Arith/IR/ArithOps.td

```diff
@@ -385,8 +398,9 @@ def Arith_DivUIOp : Arith_IntBinaryOp<"divui", [ConditionallySpeculatable]> {
     the most significant, i.e. for `i16` given two's complement representation,
     `6 / -2 = 6 / (2^16 - 2) = 0`.
 
-    Note: the semantics of division by zero is TBD; do NOT assume any specific
-    behavior.
+    Division by zero is undefined behaviour. When applied to `vector` and 
+    `tensor` values, the result is undefined if _any_ elements are divided by 
```

Comment:
My understanding is that it's not the result that is undefined, but there is undefined behavior? (i.e., it may not execute the next instructions)

Reply:
+1

Reply:
thanks mathieu, updated!